.github/actions/composite/setupGitForOSBotifyApp/action.yml

```
@@ -47,7 +47,7 @@ runs:
 
     - name: Generate a token
       id: generateToken
-      uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
+      uses: actions/create-github-app-token@v1.5.0
```

```suggestion
      uses: actions/create-github-app-token@9d97a4282b2c51a2f4f0465b9326399f53c890d4
```

I think we still prefer hash when it's passing a token

https://github.com/actions/create-github-app-token/commit/9d97a4282b2c51a2f4f0465b9326399f53c890d4

https://github.com/Expensify/App/blob/main/.github/workflows/README.md#security-rules-

---

Re-reading the rules, it looks like since this is maintained by GitHub it might be OK to use a tag? Still might prefer a hash though.

---

Oh I didn't even know we had that section. I'm fine going with hash too.